flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/flink/translator/TransformTranslator.java

```diff
@@ -85,9 +99,27 @@ public DataStream<Event> translatePostTransform(
         postTransformFunctionBuilder.addTimezone(timezone);
         postTransformFunctionBuilder.addUdfFunctions(
                 udfFunctions.stream()
-                        .map(udf -> Tuple2.of(udf.getName(), udf.getClasspath()))
+                        .map(this::udfDefToNameAndClasspathTuple)
+                        .collect(Collectors.toList()));
+        postTransformFunctionBuilder.addUdfFunctions(
+                models.stream()
+                        .map(this::modelToNameAndClasspathTuple)
                         .collect(Collectors.toList()));
         return input.transform(
                 "Transform:Data", new EventTypeInfo(), postTransformFunctionBuilder.build());
     }
+
+    private Tuple2<String, String> modelToNameAndClasspathTuple(ModelDef model) {
```

Comment:
The name of this method should be adjusted. There is not a classpath in the `ModelDef`.